### DISCUSSION TITLE
Settings sync?

### DISCUSSION THREAD

**Discussion by ryanbuening** [bluecaret/carettab#351] [Help and Support]
Should changed settings automatically sync when opened in a different browser (different computer)? I thought that was the case with 3.x, but it doesn't appear to be happening for me with 4.x.

**Comment by bluecaret** [comment]
Yes, settings should still automatically sync if it is the same browser, and you are signing into the same account on both browsers, that functionality is unchanged (uploaded wallpapers will not sync however!, or notepads if you enabled the option for a larger size).

I know sometimes the sync between computers is not the fastest thing. Also, both computers should be on v4 otherwise I'm not even sure what would happen. 

If everything is setup correctly, try making a small change on the computer with the settings you want to keep, and that might help the browser to know to sync again, and then give it a few hours, maybe a day to be sure, to see if it shows on the other machine.

If you need a faster resolution, you could do the export/import option as well.

I'll need to do some testing on this myself now, so let's leave this unanswered until either of us can confirm 👍 

Thanks!

**Reply by ryanbuening** [reply]
Sounds good. I have been manually exporting/importing for now.

**Comment by bluecaret** [comment]
It seems there was indeed a potential bug with the sync, particularly when you have settings in place on one machine, and then freshly install on another, the new machine will not recognize the new settings and will continue showing the Welcome page which if you continued on that page could result in a loss of settings. I think I got it resolved in an update going out in a couple days, v4.1.0.
